packages/cli/src/task-list.ts

```diff
@@ -71,6 +71,36 @@ export default class TaskList {
     return this.getTasks().find((task) => task.meta.taskName === taskName);
   }
 
+  /**
+   * Finds tasks by task name
+   *
+   * @method findTasks
+   * @param taskNames The name of the task to find
+   */
+  findTasks(taskNames: TaskName[]): { tasksFound: Task[]; tasksNotFound: TaskName[] } {
```

Comment:
Could we change this to use varargs instead? 

```suggestion
  findTasks(...taskNames: TaskName[]): { tasksFound: Task[]; tasksNotFound: TaskName[] } {
```